lib/api/model/model.dart

```diff
@@ -233,14 +233,6 @@ class User {
     return (value != null && value.isNotEmpty) ? value : null;
   }
 
-  static bool _readIsSystemBot(Map<dynamic, dynamic> json, String key) {
-    // This field is absent in `realm_users` and `realm_non_active_users`,
-    // which contain no system bots; it's present in `cross_realm_bots`.
```

Comment:
```
api: Drop _readIsSystemBot, relying on server 5+, FL 83+
```

This comment still applies, right? The change made in FL 83 is just that `is_cross_realm_bot` stops being a name we have to care about. The comment can go above the `defaultValue: false`, and it seems like we should keep the test that confirms that `false` is chosen if `is_system_bot` is absent.